backend/populate_backend.py

```diff
@@ -202,11 +256,39 @@ def populate():
 
 		user.save()
 
-	for i in items :
-		seller = User.objects.all(username=i['seller'])
+   for i in items : #for items
+       seller = User.objects.all(username=i['seller'])
+  
+       seller.save()
+  
+   for t in item_types:
+       item_type = ItemType.objects.get(id=req_item["type_id"])
+      
+   for o in orders: #for order 
+       payment_intent = stripe.PaymentIntent.create(
+						amount=str(int((o["price"]*o["quantity"]) * 100)),
+						currency="gbp",
+						payment_method_types=["card"],
+						application_fee_amount=str(calculate_application_fees(seller_total[o["seller_name"]], o["price"], o["quantity"]) * 100),
```

Comment:
check that this doesnt fail as the variable isn't created to get a key from 

seller total